x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_logic.ts

```diff
@@ -43,15 +96,59 @@ interface RelevanceTuningValues {
   query: string;
   unsavedChanges: boolean;
   dataLoading: boolean;
-  searchResults: object[] | null;
+  searchResults: Result[] | null;
   resultsLoading: boolean;
 }
 
+const UPDATE_SUCCESS_MESSAGE = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.relevanceTuning.messages.updateSuccess',
+  {
+    defaultMessage: 'Relevance successfully tuned. The changes will impact your results shortly.',
+  }
+);
+const DELETE_SUCCESS_MESSAGE = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.relevanceTuning.messages.deleteSuccess',
+  {
+    defaultMessage:
+      'Relevance has been reset to default values. The change will impact your results shortly.',
+  }
+);
+const RESET_CONFIRMATION_MESSAGE = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.relevanceTuning.messages.resetConfirmation',
+  {
+    defaultMessage: 'Are you sure you want to restore relevance defaults?',
+  }
+);
+const DELETE_CONFIRMATION_MESSAGE = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.relevanceTuning.messages.deleteConfirmation',
+  {
+    defaultMessage: 'Are you sure you want to delete this boost?',
+  }
+);
+
```

Comment:
I'm not a fan of chucking random miscellaneous things at the top of logic files - in this case it's definitely making an already very large file longer and harder to parse. Could we move this out to a `constants.ts` file?

Reply:
Sure. I wouldn't call it "random miscellaneous things though". They're variables that are local to and only really relevant to this file. But I agree that this file is already long. I'm happy to move them.

Reply:
309f884